src/pycricketpred/modelling.py

```diff
@@ -188,6 +188,7 @@ def evaluate_model(final_pipe, X_test, y_test, save_image_path):
     conf_mat = metrics.confusion_matrix(y_test, final_pipe.predict(X_test))
     plot_cm = metrics.ConfusionMatrixDisplay(conf_mat)
     plot_cm.plot()
+    ##could make the image name as an input of the function
```

Comment:
however, if we change this, we also need to change the tests and our original function in accordingly position.not sure if it is worthy.